### DISCUSSION TITLE
Moving Forge and Automatic to new SSD

### DISCUSSION THREAD

**Discussion by carlmoss22** [lllyasviel/stable-diffusion-webui-forge#2410] [Q&A]
Hi Guys, i want to move Forge and Automatic from C to D HDD. I copied to folders but it don't work. Automatic starts but won't generate a pic.

Forge doesn't even start.

Somebody know what to do?

I use Win 10. I deleted Venv but i think it does not rebuild without errors.

Somebody know what i have to do?

**Comment by ghost** [comment]
I recently did that, too. Removed venv, recreated and activated it, then ran `pip install -r requirements_versions.txt`.

**Reply by carlmoss22** [reply]
thank you. can you tell me what you mean with "activate venv"

**Reply by ghost** [reply]
`source venv/bin/activate`

Not sure what one has to do on windows.

**Comment by usigg3** [comment]
I did something similar. I had to format the SSD with NTFS or other decent file system as most come with FAT32 which does not support all functions needed. With a NTFS drive I could simply move it.

**Reply by carlmoss22** [reply]
Automatic is now working, Forge don't work at all. i got the error. launch.py: error: unrecognized arguments: -preset normal

**Comment by carlmoss22** [comment]
thx guys, now it's working.
